shared/chat/conversation/normal/container.js

```diff
@@ -19,11 +19,14 @@ const mapStateToProps = (state, {conversationIDKey, isPending}) => {
   const meta = Constants.getMeta(state, conversationIDKey)
   const infoPanelOpen = Constants.isInfoPanelOpen(state)
   const isSearching = state.chat2.pendingMode === 'searchingForUsers' && isPending
+  const giphyResults = state.chat2.giphyResultMap.getIn([conversationIDKey])
```

Comment:
same `getIn` -> `get` change here, can simplify the expression for `showGiphySearch` too